tests/playwright/sanity/container-grid.test.js

```diff
@@ -119,6 +119,122 @@ test.describe( 'Container tests', () => {
 			expect( isOneColumn ).toBeTruthy();
 		} );
 	} );
+
+	test( 'Test grid outline', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			parentContainer = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Arrange.
+		await test.step( 'Arrange', async () => {
+			await editor.closeNavigatorIfOpen();
+			await editor.setSelectControlValue( 'container_type', 'grid' );
+		} );
+
+		const frame = editor.getPreviewFrame(),
+			gridOutline = await frame.locator( '.e-grid-outline' ),
+			gridOutlineChildren = await frame.locator( '.e-grid-outline-item' );
+
+		let gridOutlineChildrenInitialValue = 6;
+
+		await test.step( 'Assert default outline', async () => {
+			await expect( gridOutline ).toBeVisible();
+			await expect( gridOutlineChildren ).toHaveCount( gridOutlineChildrenInitialValue );
+		} );
+
+		await test.step( 'Assert outline shut down', async () => {
```

Comment:
What means 'shut down'?

Reply:
Changed to `Turn off the outline control`